File: app/src/main/java/com/akshayltc/chime/playback/TagMediaPlayer.java
package com.akshayltc.chime.playback;

import android.content.Context;
import android.media.MediaPlayer;
import android.net.Uri;

/**
 * Created by Admin on 2017-07-16.
 */

public class TagMediaPlayer extends MediaPlayer {

    private MediaPlayer mediaPlayer;

    TagMediaPlayer(String TAG, Context mContext, String source) {
        super();
        mediaPlayer = MediaPlayer.create(mContext, Uri.parse(source));
    }



}

File: app/src/main/java/com/whitelotusapps/chime/playback/TagMediaPlayer.kt
package com.whitelotusapps.chime.playback

import android.content.Context
import android.media.MediaPlayer
import android.net.Uri

/**
 * Created by Admin on 2017-07-16.
 */
class TagMediaPlayer internal constructor(TAG: String?, mContext: Context?, source: String?) : MediaPlayer() {
    private val mediaPlayer: MediaPlayer

    init {
        mediaPlayer = create(mContext, Uri.parse(source))
    }
}